examples/quickstart/groundtruth_dataset_persistence.ipynb

```diff
@@ -24,7 +24,6 @@
    "metadata": {},
```

Comment:
<p>If this is possible with local sqlite, can you do that instead? If not, can you justify why the scale doesn't work and why you need to use Snowflake instead here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>This is mainly done for Cortex team adoption since they can all their gt datasets already in SF table - having this example is IMO useful as they can easily compare the schema in SF, instead of using local SQLite</p>

Reply:
If you want to focus on SF usage, can you move it to expositional? Don't want to be pushy with enterprise software in OSS QuickStarts

Reply:
makes sense and will do